### PR TITLE
notification logo

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,31 @@ Please open an issue if you have any questions.
 ### Small Logo
 
 [![Logo Small](gotify-logo-small.png)](gotify-logo-small.png)
+
+### Notification Logo
+
+Logo for the notification drawer was made by Choi Wong and has been optimized by
```
Comment (jmattheis) [inline README.md]: You haven't created the icon, isn't it based on the gotify icon, or where have you got it? Can it be licensed under the [Creative Commons Attribution 4.0 International Public License](http://creativecommons.org/licenses/by/4.0/)?

Comment (PanderMusubi) [reply README.md]: Friend of mine redraw it inspired on the original. Fine with the fact the Gotify organization has the rights. I will improve the text.

Comment (PanderMusubi) [reply README.md]: https://github.com/PanderMusubi/logo/commit/2c0c95135dcf914b8c502b21e1b3f0c2888da2da

File: README.md [hunk 2]
```diff
@@ -13,3 +13,32 @@ Please open an issue if you have any questions.
 ### Small Logo
 
 [![Logo Small](gotify-logo-small.png)](gotify-logo-small.png)
+
+### Notification Logo
+
+Logo for the notification drawer was based on the original logo and drawn by
+Choi Wong and has been optimized by Pander Musubi. All rights are owned by the
```
Comment (jmattheis) [inline README.md]: May I ask who is Choi Wong? is that someone you know?

Comment (PanderMusubi) [reply README.md]: Yes, a designer I know.

**Review by jmattheis — APPROVED** [review]
I've edited the readme a little, we don't have to include every size there.

**Pull request merged**